README.md

```diff
@@ -82,13 +82,20 @@ Required parameters
 Optional parameters
 -------------------
 
+All options supported in the [Notifications API specification](https://notifications.spec.whatwg.org/#dictdef-notificationoptions):
+
 * `body`: (string) - notification message body
 * `icon`: (string) - path for icon to display in notification
 * `tag`: (string) - unique identifier to stop duplicate notifications
 * `lang`: (string) - BCP 47 language tag for the notification (default: `en`)
 * `timeout`: (integer) - number of seconds to close the notification automatically
 * `silent`: (boolean) - indicates that no sounds or vibrations should be made (default: `false`)
 * `requireInteraction`: (boolean) - keep the notification open indefinitely
```

Comment:
Only including some of the options here might be confusing I think, so it should probably be all or nothing. Seeing as the link to the options available in the spec is clear and comes first, I think it makes sense to rely on those instead of duplicate things here.